Mu2eG4/src/Mu2eG4TrackingAction.cc

```diff
@@ -184,6 +191,19 @@ namespace mu2e {
 
     // Need to cast away const-ness to do this.
     const_cast<G4Track*>(trk)->SetUserInformation(ti);
+    if (_muonPreAssignedDecayProperTime>=0.0) {
+      const G4DynamicParticle* dynPart = trk->GetDynamicParticle();
+      if (dynPart->GetPDGcode() == 13) {
```

Comment:
Please use the PDGCode enum rather than the literal 13.

Reply:
Done (Also in two other cases in this file); Thanks for reminding me about it

Reply:
Thanks.